postprocessing/janus-pp-rec.c

```diff
@@ -665,6 +672,55 @@ int main(int argc, char *argv[])
 	JANUS_LOG(LOG_INFO, "Counted %"SCNu32" frame packets\n", count);
 
 	if(video) {
+		if (rtp_header_ext_id_cvo != -1) {
+			/* Parse RTP header extension now */
+			int rotation = 0, got_rotation = 0;
+			janus_pp_frame_packet *tmp = list;
+			while(tmp) {
```

Comment:
Why are you parsing in a separate while? In case there are many packets, that might take a long time, which is unneeded if we consider we already go through all the packets in other parts of the code. My suggestion would be making the CVO extraction a separate function with signature in `pp-rtp.h` and implementation in `janus-pp-rec.c`, and then invoke that function from the webm and mp4 code: this would allow to avoid repetitions (or only having it for mp4, as it was initially), and integrate it with the approach we have in use for keyframes for instance (printing when we receive a keyframe while traversing the document). Does this make sense?